kloppy/domain/models/common.py

```diff
@@ -126,6 +126,10 @@ class Position:
     def __str__(self):
         return self.name
 
+    @classmethod
+    def unknown(cls) -> "Position":
+        return cls(position_id="", name="Unknown")
+
 
 @dataclass(frozen=True)
 class Player:
```

Comment:
This class now has `initial_position`, `starting_position` and `position` attributes. I don't see immediately what's the difference between `initial_position` and `starting_position`. The implementation of the `position` attribute is also counterintuitive to me. I would expect that it returns a player's initial position, not the last position. 

I think you should be able to easily retrieve a players position at the start of the game and the list of subsequent position changes. The last position isn't more important than any other position. Hence, my suggestion would be to keep `starting_position` and make `position` identical to `starting_position` but deprecate it. I prefer `starting_position` over `initial_position`.

Reply:
Agree on the confusion naming. The `initial_position` is intended as the "position from the metadata". This can also be a position from a non-starting player.

I'll update and follow your idea. Thanks!